rs/tests/dre/utils/steps/update_subnet_type.rs

```diff
@@ -146,6 +148,85 @@ impl Step for UpdateSubnetType {
     }
 }
 
+#[derive(Clone)]
+pub struct UpdateApiBoundaryNodes {
+    pub version: String,
+}
+
+impl Step for UpdateApiBoundaryNodes {
+    fn execute(
+        &self,
+        env: ic_system_test_driver::driver::test_env::TestEnv,
+        rt: Handle,
+    ) -> anyhow::Result<()> {
+        let logger = env.logger();
+
+        // check that there are even API boundary nodes
```

Comment:
I would recommend changing all of the comments below into `info!(logger, "...")` calls such that it's easier to see what this test is doing by looking at the test output.

Reply:
Good idea! Added logs instead of the comments.